GitHubCard/index.js

```diff
@@ -27,8 +62,33 @@
     Using that array, iterate over it, requesting data for each user, creating a new card for each
     user, and adding that card to the DOM.
 */
+const eachUser = []
+const followersArray = ['https://api.github.com/users/tetondan', 'https://api.github.com/users/dustinmyers','https://api.github.com/users/justsml',
+'https://api.github.com/users/luishrd', 'https://api.github.com/users/bigknell'];
+
+followersArray.forEach(url => {
+  axios
+  .get(url)
+  .then((res) =>{
+    console.log('here is the res:', res);
```

Comment:
While `console.logs` are wonderful for debugging. Remember before sending something to production we would want to remove unnecessary `console.log` calls out so they do not pollute the console of our users and show them something that is not needed. I always try to go by a rule of thumb a teacher once told me, `Please leave your workspace like a scout would, cleaner than you left it.`  This pays off when you have to pass your project to another person, or when you come back later in time and pick your project up again and are looking through it and trying to remember what you did and do not have unneeded code just lying around.